tests/test.linter.js

```diff
@@ -1627,36 +1544,29 @@ describe('Linter.run()', () => {
     class FakeXpi extends FakeIOBase {
     }
 
-    return addonLinter.run({ _Xpi: FakeXpi, _console: fakeConsole })
-      .then(unexpectedSuccess)
-      .catch((err) => {
-        sinon.assert.calledOnce(addonLinter.handleError);
-        expect(err).toBeInstanceOf(Error);
-        expect(err.message).toContain('metadata explosion');
-      });
+    await expect(
+      addonLinter.run({ _Xpi: FakeXpi, _console: fakeConsole })
+    ).rejects.toThrowError(expectedError);
```

Comment:
`toThrow` and `toThrowError` are two aliases of the same jest assertion, let's stick to one for all the tests (and it seems that we used `toThrow` in the other tests and so it would be better to turn all the `toThrowError` into `toThrow`).